### PR TITLE
Ghostdrone construction worker tools!

### PR DESCRIPTION
**Pull request by TTerc**
## About the PR 

Adds Floor and Wall Planner to ghostdrone's list of available tools and gives ghostdrones construction visualizer vision

## Why's this needed?

Ghostdrones are tasked to maintain, repair and improve the station. Giving them Floor and Wall Planner enhances their ability to do so. It offers them more ways to improve the station to their silicon liking and also makes this role more attractive to bored ghost players.

## Changelog

```
(u)TTerc:
(+)Adds Floor and Wall Planner to ghostdrone's list of available tools
(+)Gives ghostdrone the ability to see the blueprints made with Floor and Wall Planner
```

### CODE REVIEW

File: code/mob/living/silicon/ghostdrone.dm [hunk 1]
```diff
@@ -40,6 +40,7 @@
 		ghost_drones += src
 		hud = new(src)
 		src.attach_hud(hud)
+		src.see_invisible = 8
```
Comment (ZeWaka) [inline code/mob/living/silicon/ghostdrone.dm:43]: comment on what this does

Comment (TTerc) [reply code/mob/living/silicon/ghostdrone.dm]: added the comment

**Comment by Kyle2143** [comment]
This might be a good idea, but I'm not completely convinced. The construction worker rcd thing lets you really change the walls and floors to crazy looking sprites. For construction workers, you can tell them to knock it off or beat them up to stop them, but giving that tool to ghost drones who are very nearly indestructible, could be frustrating.

**Comment by flrsh** [comment]
Ghost drones are pretty easy to beat up (or at least I think I fixed them to be). It annoys me when people make the station really ugly, though, so I'm ambivalent about this change.

**Comment by flrsh** [comment]
I'd love to hear other people's thoughts, cuz if most people think this is good, I'll merge it!

**Comment by MarkNais** [comment]
I think this is a good addition. If someone's obviously making something ugly I'd say that's a grief/ahelp issue? Maybe?
Otherwise, a "different looking station for 50 minutes" isn't too bad i.m.o.
I've also found ghostdrones to be quite frail lately.

**Comment by mordent-goonstation** [comment]
I think it might be a nice happy-medium if it was a restricted whitelist of floor tiles/walls that are typically found on the station/ship. An assortment of tile colors/arrangements (e.g. sides/corners), and some carpet stuff, for example, would cover most use-cases without being particularly grody. It'd definitely give room for more creativity, anyway.

**Comment by TTerc** [comment]
Seeing as the change would also require me (or someone) to balance the floor and wall planner itself and to remove some of the griefy textures, I am gonna close it for now
No need to keep this in PR limbo forever

**Pull request closed without merging**